active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,622 @@
+```
+![A mockup of an InfoBar in a Warning state with a close button and a message](images/Warning_TitleMessage.png)
+
+## Programmatic dismiss in InfoBar
+An InfoBar can be dismissed by the user via the close button or programmatically. 
+If the notification is required to be in view until the status is resolved and you would like to remove 
+the ability for the user to dismiss the info bar, you can set the IsUserDismissable property to false.
+
+
+By default, the close button will appear as an 'X' and the IsUserDismissable property is therefore set to true.
+
+
+XAML
+```xml
+<controls:InfoBar x:Name="NoInternetNotification"
+    Severity="Error"
+    Title="No Internet"
+    Message="Reconnect to continue working."
+    IsUserDismissable="False" />
+```
+![Mockup of an InfoBar in an Error state with no close button](images/Error_NoClose.png)
+
+## Customization: Background color and icon
+Outside of the pre-defined severity levels, the Background and IconSource properties can be set 
+to customize the icon and background color. The InfoBar will retain the assistive technology settings of 
+the severity defined, or default if none was defined.
+
+
+A custom background color can be set via the standard Background property and will override the color set by Severity.
+Please keep in mind content readability and accessibility when setting your own color.
+
+
+A custom icon can be set via the IconSource property. By default, an icon will be visible. 
+This icon can be removed by setting the IsIconVisible property to false. 
+For custom icons, the recommended icon size is 20px.
+
+
+XAML
+```xml
+<controls:InfoBar x:Name="CallRecordingNotification"
+    Title="Recording started"
+    Message="Your call has begun recording."
+    Background="{StaticResource LavenderBackgroundBrush}">
+    <controls:InfoBar.IconSource>
+        <controls:SymbolIconSource Symbol="Phone" />
+    </controls:InfoBar.IconSource>
+</controls:InfoBar>
+```
+
+![A mockup of an InfoBar in the default state with a custom background color and icon](images/Custom_IconColor.png)
+
+## Add an action button
+
+An additional action button can be added by defining your own button that inherits 
+[ButtonBase](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Controls.Primitives.ButtonBase) 
+and setting it in the ActionButton property. Custom styling will be applied to action buttons of type 
+[Button](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.Button) 
+and [HyperlinkButton](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.HyperlinkButton) 
+for consistency and accessibility. Outside of the ActionButton property, 
+additional action buttons can be added via custom content and will appear below the message.
+
+
+XAML
+```xml
+<controls:InfoBar x:Name="NoInternetNotification"
+    Severity="Error"
+    Title="No Internet"
+    Message="Reconnect to continue working.">
+    <controls:InfoBar.ActionButton>
+        <Button Content="Network Settings"/>
```

Comment:
Do I need to add a `Click` handler to this `Button`, or is there an `ActionButtonClicked` event on the `InfoBar`?

Reply:
You need to add the Click handler. There is no event for this on the InfoBar. Note that the action button is optional.

Reply:
Suggest adding it to the sample then to make it clear :)